stripe-ui-core/src/main/java/com/stripe/android/uicore/text/Html.kt

```diff
@@ -339,12 +383,9 @@ fun annotatedStringResource(
                                 )
                             }
                         }
-                        is UnderlineSpan -> {
-                            addStyle(
-                                SpanStyle(textDecoration = TextDecoration.Underline),
-                                start,
-                                end
-                            )
```

Comment:
We don't want the underline span anymore?

Reply:
ah we do, updated!